extension/src/prototypes/view/containers/ScreenSpaceCamera.tsx

```diff
@@ -62,6 +54,17 @@ export const ScreenSpaceCamera = ({ tiltByRightButton = false }: ScreenSpaceCame
       },
     ],
   };
+
+  const cameraPotion = window?.reearth?.camera?.position;
+  useEffect(() => {
```

Comment:
We might need to use `requestAnimationFrame` for this part, because rotating camera is not smooth...

Reply:
You can make a hook like below. You need to check the position of the camera in every frame.
```js
const useFrame = (cb: () => void) => {
  const [shouldStart, setShouldStart] = useState(false);
  const isCanceled = useRef(false);
  isCanceled.current = !shouldStart;
  useEffect(() => {
    let timer;
    const animate = () => {
      if(isCanceled.current) return;
      cb();
      timer = requestAnimationFrame(animate);
    };
    timer = requestAnimationFrame(animate);
    return () => cancelAnimationFrame(timer);
  }, [cb]);

  return {
    start: () => setShouldStart(true),
    stop: () => setShouldStart(false),
  };
}
```

Reply:
Since we need to correct the roll before the scene been rendered, it requires to be implemented in reearth.